test/CoreTest.cpp

```diff
@@ -503,11 +519,13 @@ TEST_CASE( "wrapper_test", "[core]" )
     mrc2.loadChunk({0}, {1}, loadData);
     o.flush();
     REQUIRE(loadData[0] == value);
-    value = 43;
+    value = 43.;
     mrc2.makeConstant(value);
-    o.iterations[4].meshes["E"]["y"].loadChunk({0}, {1}, loadData);
+    std::array< double, 1 > moreData = { 112233. };
+    auto pppp = loadRaw(moreData);
```

Comment:
note: we need an `lvalue` since `loadChunk` needs a `ref`

we could also provide a pure `*` interface that does this wrapping.